src/createServer.js

```diff
@@ -1,3 +1,42 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { checkErrors } = require('./validation');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, 'http://localhost:5700');
```

Comment:
The hardcoded localhost URL in URL constructor might be a problem when deploying on a different server or port. It would be better to take hostname and port from the request headers or use environment variables.
```suggestion
    const normalizedUrl = new URL(req.url, `http:${req.headers.host}`);
```